__tests__/components/ui/Certificate.test.tsx

```diff
@@ -0,0 +1,28 @@
+import Certificate from "@/components/ui/Certificate";
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+
+jest.mock("@/icons/certificates/Celo.svg", () => "CeloIcon");
+jest.mock("@/icons/certificates/Algoland.svg", () => "AlgolandIcon");
+jest.mock("@/icons/certificates/Blockchain.svg", () => "BlockChainIcon");
+jest.mock("@/icons/certificates/Eternity.svg", () => "EthIcon");
+jest.mock("@/icons/certificates/Near.svg", () => "NearIcon");
+jest.mock("@/icons/certificates/Icp.svg", () => "IcpIcon");
+jest.mock("@/icons/certificates/Ai.svg", () => "AiIcon");
+jest.mock("@/icons/certificates/Solidity.svg", () => "SolidityIcon");
+jest.mock("@/icons/certificates/Sui.svg", () => "SuiIcon");
```

Comment:
This is not right, it should already be mocked globally, if not yet, please apply a transformer, so we dont need to mock all the time we want to test a component with an SVG